bootstrap/src/integration-test/resources/log4j.properties

```diff
@@ -0,0 +1,4 @@
+log4j.rootLogger=INFO, X
+log4j.appender.X=org.apache.log4j.ConsoleAppender
+log4j.appender.X.layout=org.apache.log4j.PatternLayout
+log4j.appender.X.layout.ConversionPattern=[%d{dd.MM.yyyy HH:mm:ss,SSS}] bootstrap [%-5p] [%t] %m%n
```

Comment:
May I ask you to use xml, json or yaml configuration instead of .properties?

Reply:
I think there should be created a separate refactoring task since we have the same properties file in all the modules

Reply:
All modules of notary, or for entire D3?

Reply:
![Screen Shot 2019-04-05 at 10 58 59](https://user-images.githubusercontent.com/17858062/55612834-d65ffc80-5791-11e9-8602-310bb50ec985.png)